### PR TITLE
AzureStack AzureBridge Admin

### PR DESCRIPTION
**Pull request by deathly809**
Initial commit for AzureStack AzureBridge admin specs.

### PR information
- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [X] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [X] If applicable, the PR references the bug/issue that it fixes.
- [X] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X] My spec meets the review criteria:
  - [X] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [X] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/azsadmin/resource-manager/azurebridge/Microsoft.AzureBridge.Admin/2016-01-01/Activation.json [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+                ],
+                "x-ms-pageable": {
+                    "nextLinkName": "nextLink"
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroup}/providers/Microsoft.AzureBridge.Admin/activations/{activationName}": {
+            "get": {
+                "x-ms-examples": {
+                    "Return product name.": {
```
Comment (bganapa) [inline specification/azsadmin/resource-manager/azurebridge/Microsoft.AzureBridge.Admin/2016-01-01/Activation.json]: activation name

File: specification/azsadmin/resource-manager/azurebridge/Microsoft.AzureBridge.Admin/2016-01-01/Activation.json [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+                "produces": [
+                    "application/json"
+                ],
+                "consumes": [
+                    "application/json"
+                ]
+            },
+            "delete": {
+                "x-ms-examples": {
+                    "Return product name.": {
```
Comment (bganapa) [inline specification/azsadmin/resource-manager/azurebridge/Microsoft.AzureBridge.Admin/2016-01-01/Activation.json]: product -> activation

File: specification/azsadmin/resource-manager/azurebridge/Microsoft.AzureBridge.Admin/2016-01-01/Activation.json [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+                ],
+                "consumes": [
+                    "application/json"
+                ]
+            }
+        }
+    },
+    "definitions": {
+        "ActivationResource": {
+            "description": "Published product resource.",
```
Comment (bganapa) [inline specification/azsadmin/resource-manager/azurebridge/Microsoft.AzureBridge.Admin/2016-01-01/Activation.json]: description product to activation

**Review by bganapa — APPROVED** [review]

File: specification/azsadmin/resource-manager/azurebridge/Microsoft.AzureBridge.Admin/2016-01-01/Activation.json (hunk 2 above)
Comment (deathly809) [reply specification/azsadmin/resource-manager/azurebridge/Microsoft.AzureBridge.Admin/2016-01-01/Activation.json]: Done

File: specification/azsadmin/resource-manager/azurebridge/Microsoft.AzureBridge.Admin/2016-01-01/Activation.json (hunk 3 above)
Comment (deathly809) [reply specification/azsadmin/resource-manager/azurebridge/Microsoft.AzureBridge.Admin/2016-01-01/Activation.json]: Done

File: specification/azsadmin/resource-manager/azurebridge/Microsoft.AzureBridge.Admin/2016-01-01/Activation.json (hunk 1 above)
Comment (deathly809) [reply specification/azsadmin/resource-manager/azurebridge/Microsoft.AzureBridge.Admin/2016-01-01/Activation.json]: Done

**Comment by sergey-shandar** [comment]
@ravbhatnagar new API `Microsoft.AzureBridge.Admin/2016-01-01/Activation.json`

**Comment by ravbhatnagar** [comment]
Azure Stack RP. I am not sure ARM needs to review this. Please let me know if otherwise. Removing the label for now.

**Comment by deathly809** [comment]
@ravbhatnagar  Just for Azure Stack, Azure does not have an admin interface.

File: specification/azsadmin/resource-manager/azurebridge/Microsoft.AzureBridge.Admin/2016-01-01/DownloadedProduct.json [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+                "responses": {
+                    "202": {
+                        "description": "Accepted",
+                        "schema": {
+                            "$ref": "#/definitions/DownloadedProductResource"
+                        }
+                    },
+                    "404": {
+                        "description": "Not Found"
+                    }
```
Comment (sergey-shandar) [inline specification/azsadmin/resource-manager/azurebridge/Microsoft.AzureBridge.Admin/2016-01-01/DownloadedProduct.json]: long running operation should also have status code 200 or 204.

File: specification/azsadmin/resource-manager/azurebridge/Microsoft.AzureBridge.Admin/2016-01-01/Product.json [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
+                    },
+                    {
+                        "$ref": "AzureBridge.json#/parameters/ProductNameParameter"
+                    },
+                    {
+                        "$ref": "AzureBridge.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "202": {
```
Comment (sergey-shandar) [inline specification/azsadmin/resource-manager/azurebridge/Microsoft.AzureBridge.Admin/2016-01-01/Product.json]: Same here. It should also have 200 or 204.

**Review by sergey-shandar — CHANGES_REQUESTED** [review]

File: specification/azsadmin/resource-manager/azurebridge/Microsoft.AzureBridge.Admin/2016-01-01/DownloadedProduct.json (hunk 4 above)
Comment (deathly809) [reply specification/azsadmin/resource-manager/azurebridge/Microsoft.AzureBridge.Admin/2016-01-01/DownloadedProduct.json]: Done.

File: specification/azsadmin/resource-manager/azurebridge/Microsoft.AzureBridge.Admin/2016-01-01/Product.json (hunk 5 above)
Comment (deathly809) [reply specification/azsadmin/resource-manager/azurebridge/Microsoft.AzureBridge.Admin/2016-01-01/Product.json]: Done.

**Comment by deathly809** [comment]
@sergey-shandar I have moved the modules to the preview folder.

**Review by sergey-shandar — APPROVED** [review]

**Pull request merged**